kafka-rest/src/main/java/io/confluent/kafkarest/resources/v3/TopicsResource.java

```diff
@@ -140,6 +141,11 @@ public void createTopic(
       @Suspended AsyncResponse asyncResponse,
       @PathParam("clusterId") String clusterId,
       @Valid CreateTopicRequest request) {
+
+    if (request == null) {
+      throw Errors.invalidPayloadException("Null input provided. Data is required.");
```

Comment:
I think something along the lines of "request body is empty" instead of null input.